auth/src/main/java/com/firebase/ui/auth/ui/email/RegisterEmailFragment.java

```diff
@@ -210,6 +213,39 @@ public void onClick(View view) {
         });
     }
 
+    private void setUpPrivacyPolicy() {
+        if (TextUtils.isEmpty(mHelper.getFlowParams().privacyPolicyUrl)) {
+            return;
+        }
+
+        ForegroundColorSpan foregroundColorSpan =
+                new ForegroundColorSpan(ContextCompat.getColor(getContext(), R.color.linkColor));
+
+        String link = getString(R.string.privacy_policy);
+        SpannableStringBuilder spannableStringBuilder = new SpannableStringBuilder(link);
+        int start = mAgreementText.length();
+        spannableStringBuilder.setSpan(foregroundColorSpan, start, start + link.length(), 0);
+
+        mAgreementText.append(" and the ");
```

Comment:
Instead of doing this which isn't translatable, could we instead combine the `setUpTermsOfService` and `setUpPrivacyPolicy` methods to make the two clickable bits of text in one go? This should simplify things and fix the bug below.